i18n/zh/docusaurus-plugin-content-docs/version-v1.3/tutorials/consume-cloud-services.md

```diff
@@ -69,7 +69,7 @@ KubeVela支持的云资源请见[列表](../end-user/components/cloud-services/c
 
 根据你的需要设置上述参数，完成应用程序的创建，然后[部署应用程序](../how-to/dashboard/application/deploy-application)。
 
-##查看云资源创建状态
+## 查看云资源创建状态
```

Comment:
great,  please also fix the docs here: https://github.com/oam-dev/kubevela.io/blob/main/i18n/zh/docusaurus-plugin-content-docs/current/tutorials/consume-cloud-services.md

Then the versioned docs won't be overrided.

Reply:
Ok, I fixed it already.